packages/api-reference/src/hooks/useMultipleDocuments.ts

```diff
@@ -141,18 +140,18 @@ export const useMultipleDocuments = ({
   const selectedConfiguration = computed(() => {
     // Multiple sources
     if (configuration.value && isConfigurationWithSources(configuration.value)) {
-      return apiReferenceConfigurationSchema.parse({
+      return {
         ...configuration.value,
         ...configuration.value?.sources?.[selectedDocumentIndex.value],
         ...availableDocuments.value[selectedDocumentIndex.value],
-      })
+      }
```

Comment:
why do we need this change?

Reply:
we don't want to parse here, we want to only parse in one place so we currently do it in the apireferencelayout (just because of org) but eventually we will bring it back out here. Otherwise we get defaults and what not

Reply:
Now I can't remember exactly what issue it was solving but there was one!

Reply:
I can remove this change for now and we can bring it back if we see it, this is what its breaking the tests